homeassistant/components/alexa/entities.py

```diff
@@ -404,6 +405,9 @@ def interfaces(self):
         if supported & media_player.SUPPORT_SELECT_SOURCE:
             yield AlexaInputController(self.entity)
 
+        if supported & media_player.const.SUPPORT_PLAY_MEDIA:
+            yield AlexaChannelController(self.entity)
+
```

Comment:
Out of curiosity, what integration are you using/testing this with? 

Unfortunately, it looks like there need to be some additions to the media_player entity model to determine support for channel equipped media players, otherwise there is a potential to expose this controller to all media_player entities that support `SUPPORT_PLAY_MEDIA`.